pnc_cli/swagger_client/apis/bpm_api.py

```diff
@@ -430,7 +430,7 @@ def start_r_creation_task_with_single_url(self, body, **kwargs):
                                             body=body_params,
                                             post_params=form_params,
                                             files=files,
-                                            response_type=int,
+                                            response_type=None,
```

Comment:
this should remain int

Reply:
I actually did check the backend code on that change (and also the moving funcs and fields around).

For the moving thing, I guess it's ok cause subsequent swagger api regeneration won't do those changes again? I dunno.

I'll double check if the response type should stay int or None though

Reply:
I'm not sure it won't change when regenerating again. I think I tried that once before but second regeneration ended up differently (but can't be sure if the generator version was the same in both cases or if the api on PNC changed)